google/cloud/spanner_v1/streamed.py

```diff
@@ -190,6 +190,19 @@ def one_or_none(self):
         except StopIteration:
             return answer
 
+    def fetch_as_dictionary_list(self):
+        rows = []
+        for row in self:
```

Comment:
What happens if rows are not fully processed when this method is called ?

Reply:
There will be a delay in returning the result in those cases but always the full result will be returned